src/porepy/grids/simplex.py

```diff
@@ -265,7 +265,13 @@ def __init__(
         face_nodes = face_nodes.reshape((3, 4 * num_cells), order="F")
         sort_ind = np.squeeze(np.argsort(face_nodes, axis=0))
         face_nodes_sorted = np.sort(face_nodes, axis=0)
-        face_nodes, _, cell_faces = setmembership.unique_columns_tol(face_nodes_sorted)
+
+        # IMPLEMENTATION NOTE: Do not use the order-preserving option to comply with
+        # tacit, hard-coded assumptions that are made on the order of nodes, faces, and
+        # cells in the test `TestMVEMRHS.test_3d_isotropic_permeability`.
```

Comment:
I guess we can remove the comment here? Also, did you have to change the reference value in the test that is referred to?